dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Account/OfficialName/Confirm.cshtml.cs

```diff
@@ -40,13 +40,16 @@ public class Confirm : PageModel
     [FromQuery(Name = "fileName")]
     public string? FileName { get; set; }
 
+    [FromQuery(Name = "fileId")]
+    public string? FileId { get; set; }
+
     public void OnGet()
     {
     }
 
     public async Task<IActionResult> OnPost()
     {
-        var sasUri = await _dqtEvidenceStorage.GetSasConnectionString(FileName!, SasTokenValidMinutes);
+        var sasUri = await _dqtEvidenceStorage.GetSasConnectionString(FileId!, SasTokenValidMinutes);
 
         var teacherNameChangeRequest = new TeacherNameChangeRequest()
```

Comment:
Should this nameChangeRequest be using
EvidenceFileName = FileName!
or
EvidenceFileName = FileId!
?
The former is the user chosen name but it is uploaded to blob storage as e.g. {FileId}.jpg

Reply:
Good spot

Reply:
Not sure if you got this already, but I was referring to the TeacherNameChangeRequest. I think we may also want to change FileName -> FileId there as well.